lib/chef/data_bag.rb

```diff
@@ -70,11 +70,11 @@ def to_json(*a)
     end
 
     def chef_server_rest
-      @chef_server_rest ||= Chef::REST.new(Chef::Config[:chef_server_url])
+      @chef_server_rest ||= Chef::ServerAPI.new(Chef::Config[:chef_server_url])
```

Comment:
kind of vaguely curious why we don't have a require change up top for all these...  not sure if the answer to that question will make me sad though...